test/unit/spec/v2/view-builder/internals/FormInputFactory_spec.js

```diff
@@ -66,26 +67,151 @@ describe('v2/view-builder/internals/FormInputFactory', function () {
   });
 
   it('handles authenticatorVerify Select type', function () {
-    const opt = {
-      type: 'authenticatorVerifySelect',
-      options: [
+    const input = {
+      "name":"authenticator",
+      "type":"authenticatorVerifySelect",
+      "options":[
         {
-          'label': 'Okta Password',
-          'authenticatorType': 'password',
-          'value': {
-            id: 'autwa6eD9o02iBbtv0g3'
-          }
+          "label":"Okta Password",
+          "value":{
+            "id":"aidwboITrg4b4yAYd0g3"
+          },
+          "relatesTo":{
+            "displayName":"Okta Password",
+            "type":"password",
+            "id":"autwa6eD9o02iBbtv0g1",
+            "authenticatorId":"aidwboITrg4b4yAYd0g3"
+          },
+          "authenticatorType":"password"
         },
         {
-          'label': 'Okta E-mail',
-          'authenticatorType': 'email',
-          'value': {
-            id: 'autwa6eDxxx2iBbtv0g3'
-          }
+          "label":"Security Key or Biometric Authenticator",
```

Comment:
I am using a different input object here and then in `handles authenticatorEnroll Select type` to cover these scenarios - 
* If there are multiple Security Key authenticator, Verify filters them to have only one.
* There could be multiple phone authenticators (voice and sms) during verification.